src/main/java/seedu/address/logic/commands/WhitelistListCommand.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.ArgumentPredicateToFail;
+import seedu.address.model.person.ClientStatus;
+
+/**
+ * Lists out all whitelisted clients
+ */
+public class WhitelistListCommand extends WhitelistCommand {
+
+    /**
+     * Instantiates a {@code WhitelistListCommand} object.
+     *
+     * @param argPredToFail an argument predicate that should fail
+     */
+    public WhitelistListCommand() {
+        // this is irrelevant but necessary
+        super(Index.fromZeroBased(0), new ClientStatus("old"));
+    }
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+
+        // here, the predicate dictates that the client status should be blacklisted;
+        // hence if the predicate fails (ensured by the ArgumentPredicateToFail object),
+        // the client is not blacklisted => they are whitelisted
+        model.updateFilteredPersonList(ArgumentPredicateToFail.getNotBlacklistedPredicate());
```

Comment:
Just for my understanding, how come there is a different execution for whitelistcommand and blacklistcommand? Couldn't it be easier if there was a predicate 'getBlacklistedPredicate' that kept things consistent and reduces Blacklist's dependence on other code? That being said, I think this still works 👍

Reply:
Yep, I will be decoupling the `blacklist` command to make it work more like the `whitelist` command later on, that was just an easy temporary fix.